staging/src/k8s.io/apiserver/pkg/cel/environment/base_test.go

```diff
@@ -41,3 +45,90 @@ func BenchmarkLoadBaseEnvDifferentVersions(b *testing.B) {
 		MustBaseEnvSet(version.MajorMinor(1, uint(i)))
 	}
 }
+
+func TestLibraryCoverage(t *testing.T) {
+	libs := library.RegisteredLibraries()
```

Comment:
do we really need to export RegisteredLibraries and call it here?

seems like we can iterate over every version between `min(baseOpts[*].IntroducedVersion, baseOpts[*].RemovedVersion)` to `max(baseOpts[*].IntroducedVersion, baseOpts[*].RemovedVersion)` and make sure that once a lib is introduced, it isn't dropped, without needing `RegisteredLibraries()`

Reply:
This is better, thanks. I'll drop the new libraries.go file in favor of this.

Reply:
I upstreamed https://github.com/google/cel-go/pull/822 to make this cleaner. I'll pick it up in this PR shortly.